gazebo_client:
  joint_names:
    - single_leg::front_left_hip_yaw
    - single_leg::front_left_hip_pitch
    - single_leg::front_left_knee_joint     

  joint_config: [
    45, -20, 90
  ]

  joint_velocity_controller:
    p_gains: [
      200, 200, 200
    ]
    i_gains: [
      0, 0, 0
    ]
    d_gains: [
      20, 20, 20
    ]
  joint_position_controller:
    p_gains: [
      500, 500, 500
    ]
    i_gains: [
      5, 5, 5
    ]
    d_gains: [
      50,50,50
    ]
